### ISSUE TITLE
Downloads broken since v2.13

### ISSUE THREAD

**Issue by awillinger** [Starblaster64/Vs-Saxton-Hale-2#242]
Ever since the release of v2.13, the downloads for the default hale models (like CBS, Bunny, etc.) no longer work. Additionally, none of the sounds are precached, causing them to simply not play.

The culprit seems to be the commented out code in handler.sp: https://github.com/VSH2-Devs/Vs-Saxton-Hale-2/blame/develop/addons/sourcemod/scripting/modules/handler.sp, where previously, several AddXToDownloads calls were present which took care of adding the downloads. It seems like since that change, all downloads have to be added to the vsh2.cfg, but this was not done while refactoring (lost about an hour trying to figure out while nothing got downloaded...).

Can you please either uncomment the code again or add the downloads into the default config (maybe commented out)?

**Comment by kevyonan** [comment]
just making sure, did you add the default boss configs?

**Comment by awillinger** [comment]
Yes, I uploaded the entire configs directory. The bosses themselves work just fine, they are just missing their models. Looking at handler.sp, the old code to add the files to the downloads table (AddHaleToDownloads for example) are commented out.

**Comment by kevyonan** [comment]
> Yes, I uploaded the entire configs directory. The bosses themselves work just fine, they are just missing their models. Looking at handler.sp, the old code to add the files to the downloads table (AddHaleToDownloads for example) are commented out.
> […](#)
> On 4 Jan 2024, 02:54, at 02:54, Kevin Yonan ***@***.***> wrote: just making sure, did you add the default boss configs? -- Reply to this email directly or view it on GitHub: [#242 (comment)](https://github.com/VSH2-Devs/Vs-Saxton-Hale-2/issues/242#issuecomment-1876203897) You are receiving this because you authored the thread. Message ID: ***@***.***>

because the default boss configs are supposed to handle that instead of handler.sp
